tests/model_registry/rest_api/mariadb/conftest.py

```diff
@@ -0,0 +1,122 @@
+from typing import Any, Generator
+from kubernetes.dynamic import DynamicClient
+import pytest
+
+from ocp_resources.maria_db import MariaDB
+from ocp_resources.mariadb_operator import MariadbOperator
+from utilities.constants import OPENSHIFT_OPERATORS, MARIADB
+from utilities.general import generate_random_name
+
+from tests.model_registry.constants import (
+    OAUTH_PROXY_CONFIG_DICT,
+    MODEL_REGISTRY_STANDARD_LABELS,
+    MR_INSTANCE_NAME,
+)
+from ocp_resources.secret import Secret
+from simple_logger.logger import get_logger
+from ocp_resources.model_registry_modelregistry_opendatahub_io import ModelRegistry
+from utilities.mariadb_utils import wait_for_mariadb_pods
+
+LOGGER = get_logger(name=__name__)
+
+
+@pytest.fixture(scope="class")
+def mariadb_str() -> str:
+    return generate_random_name(prefix=MARIADB, length=4)
+
+
+@pytest.fixture(scope="class")
+def mariadb_dict(mariadb_str: str) -> dict[str, Any]:
+    return {
```

Comment:
I think this would need to be parametrized from the beginning if we are aiming to move that way